reconcile/test/test_terraform_tgw_attachments.py

```diff
@@ -0,0 +1,302 @@
+import pytest
+
+import reconcile.terraform_tgw_attachments as integ
+
+
+@pytest.fixture
+def account_tgw_connection():
+    return {
+        "name": "account_tgw_connection",
+        "provider": "account-tgw",
+        "manageRoutes": True,
+        "account": {
+            "name": "tgw_account",
+            "uid": "a-uid",
+            "terraformUsername": "tf-user",
+        },
+        "assumeRole": None,
+        "cidrBlock": "172.16.0.0/16",
+        "deleted": False,
+    }
+
+
+@pytest.fixture
+def account_vpc_connection():
+    return {
+        "name": "account_vpc_connection",
+        "provider": "account-vpc",
+    }
+
+
+@pytest.fixture
+def cluster_with_tgw_connection(account_tgw_connection):
+    return {
+        "name": "cluster_with_tgw_connection",
+        "ocm": {"name": "cluster_with_tgw_connection-ocm"},
+        "spec": {
+            "region": "us-east-1",
+        },
+        "network": {"vpc": "10.0.0.0/16"},
+        "peering": {
+            "connections": [
+                account_tgw_connection,
+            ]
+        },
+    }
+
+
+@pytest.fixture
+def cluster_with_vpc_connection(account_vpc_connection):
```

Comment:
nit: would it make sense to extract these fixtures into a builder fixture instead, e.g., https://github.com/app-sre/qontract-reconcile/blob/master/reconcile/test/conftest.py#L45-L75 or https://github.com/app-sre/qontract-reconcile/blob/master/reconcile/test/saas_auto_promotions_manager/merge_request_manager/merge_request_manager/conftest.py#L69-L80?

This could then later be easily adjusted to use the [gql_class_factory](https://github.com/app-sre/qontract-reconcile/blob/master/reconcile/test/conftest.py#L172) once we use typed queries for this integration.

Reply:
cool, I'll try this today

Reply:
I'll merge this pr then create a new one for refactor